scripts/codes/business-investment-prices.json

```diff
@@ -0,0 +1,3 @@
+{"_id":"a7716708-183e-4ceb-991d-ea8cfc57b092","code":"L","label":"Chain linked volume","links":{"code_list":{"id":"business-investment-prices","href":"http://localhost:22400/code-lists/business-investment-prices"},"self":{"id":"L","href":"http://localhost:22400/code-lists/business-investment-prices/codes/L"}}}
```

Comment:
Again, recognizing 'because the business' as a reason, could these codes be more descriptive? `chain-linked-volume` etc?

Reply:
the codes are SDMX sourced which is all about the efficiency and very, very much not about user friendliness. Those codes are reusable definitions though, and can (and should) appear in other datasets going froward. Would be very wary of changing them.

Reply:
absolutely fair reason not to do it 👍